### PR TITLE
fixing joint names, image border to white

### PR DESCRIPTION
**Pull request by Lawproto**
As pointed out F2F by @fiorisi, the `sim_ecub_` prefix for the is Creo (sim model) specific and should not be part of the names for URDF.
I also catch the occasion to remove an unwanted orange border to one of the images.

@pattacini

### CODE REVIEW

File: docs/hands/hands_mk5.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+|               |        | l_middle_dist |  ROT | l_hand_middle_1    | l_hand_middle_2    | M2 in fig.      |
+|               |        | l_ring_prox   |  ROT | l_hand_palm        | l_hand_ring_1      | R1 in fig.      |
+|               |        | l_ring_dist   |  ROT | l_hand_ring_1      | l_hand_ring_2      | R2 in fig.      |
+|               |        | l_pinkie_prox |  ROT | l_hand_palm        | l_hand_pinkie_1    | P1 in fig.      |
+|               |        | l_pinkie_dist |  ROT | l_hand_pinkie_1    | l_hand_pinkie_2    | P2 in fig.      |
 
 
 <p align="center">
-  <img  src=    "../img/pitches.png"
+  <img  src=    "./img/pitches.png"
```
Comment (pattacini) [inline docs/hands/hands_mk5.md]: This is problematic as pointed out in #238.

**Comment by Lawproto** [comment]
Also part of https://github.com/icub-tech-iit/dev/issues/597

**Review by pattacini — APPROVED** [review]

File: docs/hands/hands_mk5.md (hunk 1 above)
Comment (pattacini) [reply docs/hands/hands_mk5.md]: Already fixed!

**Comment by pattacini** [comment]
Remember not to put links to private repos into public repos.

**Pull request merged**